pkg/limits/partition_manager.go

```diff
@@ -50,23 +45,23 @@ func (m *PartitionManager) Has(partition int32) bool {
 	return ok
 }
 
-// List returns a map of all assigned partitions and their last updated timestamps.
+// List returns a map of all assigned partitions and the timestamp of when
+// each partition was assigned.
```

Comment:
should this say "when each partition was most recently assigned" for when Assign updates the timestamp of a previous assignment?